packages/runtime/src/useCases/transport/files/UploadOwnFile.ts

```diff
@@ -53,15 +53,6 @@ class Validator extends SchemaValidator<UploadOwnFileValidatableRequest> {
             );
         }
 
-        if (input.expiresAt && CoreDate.from(input.expiresAt).isSameOrBefore(CoreDate.utc())) {
-            validationResult.addFailure(
-                new ValidationFailure(
-                    RuntimeErrors.general.invalidPropertyValue(`'${nameof<UploadOwnFileValidatableRequest>((r) => r.expiresAt)}' must be in the future`),
-                    nameof<UploadOwnFileValidatableRequest>((r) => r.expiresAt)
-                )
-            );
-        }
-
```

Comment:
I don't like, that you squeezed in a major refactoring into this PR. This has nothing to do with the actual PR.